trieste/models/gpflux/models.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2021 The Trieste Contributors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from typing import Optional
+
+import gpflow
+import gpflux
+import tensorflow as tf
+import tensorflow_probability as tfp
+from gpflux.models import DeepGP
+from gpflux.models.deep_gp import sample_dgp
+from gpflux.layers import GPLayer
+import warnings
+
+from ...data import Dataset
+from ...types import TensorType
+from ...utils import jit
+from ..interfaces import TrainableProbabilisticModel
+from ..optimizer import TFOptimizer
+from .interface import GPfluxPredictor
+
+
+class VanillaDeepGP(GPfluxPredictor, TrainableProbabilisticModel):
```

Comment:
not sure if `Vanilla` is the best name, but well, you do have some artistic freedom here :)

Reply:
I'm happy to take suggestions, I'm not by any means fixed on Vanilla :)